app/views/notifications/index.html.erb

```diff
@@ -4,7 +4,7 @@
     <%= render @notifications %>
   <% else %>
     <div class="mt-8">
-      <%= render "empty_state" %>
+      <%= render EmptyStateComponent.new(title: t("notifications.empty_state.title"), body: t("notifications.empty_state.body"), icon: "icons/outline/bell.svg") %>
```

Comment:
Let's move the translations so they aren't so long to access. For example,

```yaml
en:
  notifications:
    index:
      empty_state:
        title:
        body:
```

Then we can use `t(".empty_state.title")` and `t(".empty_state.body")` here.

Reply:
I went with this approach first, but wasn't sure if there was previous history with how it was placed in `en.yml` - will change this!